tests/test_kojiservice.py

```diff
@@ -142,3 +142,19 @@ def test_get_bundle_csv_unavailable(mock_get, mock_koji, mock_log):
devel-0:8.10.5-115.fc35.s390x\n      - ghc-ghc-compact-0:0.1.0.0-115.fc35.x86_64\n      - ghc-bytestring-0:0.10.12.0-115.fc35.armv7hl\n      - ghc-unix-prof-0:2.7.2.2-115.fc35.aarch64\n      - ghc-bytestring-prof-0:0.10.12.0-115.fc35.s390x\n      - ghc-0:8.10.5-115.fc35.ppc64le\n      - ghc-ghc-devel-0:8.10.5-115.fc35.x86_64\n      - ghc-stm-prof-0:2.5.0.1-115.fc35.ppc64le\n      - ghc-array-0:0.5.4.0-115.fc35.aarch64\n      - ghc-deepseq-prof-0:1.4.4.0-115.fc35.aarch64\n      - ghc-ghc-devel-0:8.10.5-115.fc35.aarch64\n      - ghc-process-devel-0:1.6.9.0-115.fc35.x86_64\n      - ghc-template-haskell-devel-0:2.16.0.0-115.fc35.s390x\n      - ghc-libiserv-0:8.10.5-115.fc35.i686\n      - ghc-time-doc-0:1.9.3-115.fc35.noarch\n      - ghc-ghc-heap-devel-0:8.10.5-115.fc35.i686\n      - ghc-exceptions-devel-0:0.10.4-115.fc35.aarch64\n      - ghc-transformers-0:0.5.6.2-115.fc35.x86_64\n      - ghc-ghc-boot-0:8.10.5-115.fc35.x86_64\n      - ghc-mtl-prof-0:2.2.2-115.fc35.s390x\n      - ghc-deepseq-devel-0:1.4.4.0-115.fc35.ppc64le\n      - ghc-pretty-devel-0:1.1.3.6-115.fc35.armv7hl\n      - ghc-template-haskell-prof-0:2.16.0.0-115.fc35.aarch64\n      - ghc-deepseq-doc-0:1.4.4.0-115.fc35.noarch\n      - ghc-time-0:1.9.3-115.fc35.armv7hl\n      - ghc-ghc-heap-0:8.10.5-115.fc35.i686\n      - ghc-ghc-prof-0:8.10.5-115.fc35.aarch64\n  dependencies:\n  - buildrequires:\n      ghc: [8.10]\n      platform: [f36]\n    requires:\n      platform: [f36]\n  references:\n    community: https://www.haskell.org/ghc\n    documentation: https://wiki.haskell.org/GHC\n    tracker: https://gitlab.haskell.org/ghc/ghc/issues\n  profiles:\n    all:\n      description: standard installation\n      rpms:\n      - ghc\n      - ghc-doc\n      - ghc-prof\n    default:\n      description: standard installation\n      rpms:\n      - ghc\n    minimal:\n      description: just compiler and base\n      rpms:\n      - ghc-base-devel\n    small:\n      description: compiler with main core libs\n      rpms:\n      - ghc-devel\n  components:\n    rpms:\n      ghc:\n        rationale: compiler\n        repository: git+https://src.fedoraproject.org/rpms/ghc\n        cache: https://src.fedoraproject.org/repo/pkgs/ghc\n        ref: 9.2\n        buildorder: 1\n        arches: [aarch64, armv7hl, i686, ppc64le, s390x, x86_64]\n...\n', 'name': 'ghc', 'stream': '9.2', 'module_build_service_id': 13274, 'version': '3620211101111632', 'context': 'd099bf28', 'content_koji_tag': 'module-ghc-9.2-3620211101111632-d099bf28'}}}, 'id': 1850907, 'name': 'ghc', 'nvr': 'ghc-9.2-3620211101111632.d099bf28', 'owner_id': 56, 'owner_name': 'petersen', 'package_id': 1853, 'package_name': 'ghc', 'release': '3620211101111632.d099bf28', 'source': 'https://src.fedoraproject.org/modules/ghc.git?#5405c8a084c04476a956f1b3e49a733b888066a5', 'start_time': '2021-11-01 13:48:17+00:00', 'start_ts': 1635774497.0, 'state': 1, 'task_id': None, 'version': '9.2', 'volume_id': 0, 'volume_name': 'DEFAULT', 'cg_name': 'module-build-service'}
```

Comment:
Could you put this on multiple lines for readability? Also you can remove fields that you don't need for the test, since this is a test.

Reply:
I was thinking to save the actual data and maybe in future we can reference different attributes other than name and stream.
But sure, let's make it more readable.